04/index.html

```diff
@@ -1,30 +1,43 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link rel="stylesheet" href="styles.css/reset.css">
+    <link rel="stylesheet" href="styles.css/styles.css">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
 </head>
+
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
-                </ul>
-            </li>
-            <li>
-                <a href="#">kontakt</a>
-            </li>
-        </ul>
-    </nav>
+
+    <header>
+        <nav class="nav">
+            <input type="checkbox" class="nav-toggle">
+            <div class="nav__hamburger">
+                <div class="hamburger-line"></div>
+            </div>
+            <ul class="nav__list">
+                <li class="list__item">
+                    <a class="list__item--link" href="#">start</a>
```

Comment:
Modyfikator nie może sam występować, prawidłowy zapis to: `class="list__item list__item--link"`
Tutaj znajdziesz więcej informacji: https://devmentor.pl/b/metodologia-bem-w-css-i-sass